connector_test.go

```diff
@@ -149,25 +141,28 @@ func bindMountHelper(t *testing.T, options string) {
 	// Note: If it ends up with length zero buffer, restarting the VM may help:
 	// https://stackoverflow.com/questions/71977532/podman-mount-host-volume-return-error-statfs-no-such-file-or-directory-in-ma
 	readBuffer := readOutputUntil(t, container, string(fileContent))
-	assert.Contains(t, string(readBuffer), string(fileContent))
+	assert.Equals(t, strings.Contains(string(readBuffer), string(fileContent)), expectedPass)
 }
 
 func TestBindMount(t *testing.T) {
-	scenarios := map[string]string{
-		"ReadOnly":   ":ro",
-		"Multiple":   ":ro,noexec",
-		"No options": "",
+	type param struct {
+		option       string
+		expectedPass bool
+	}
+	scenarios := map[string]param{
+		"ReadOnly":   param{":ro", true},
+		"Multiple":   param{":ro,noexec", true},
+		"No options": param{"", true},
```

Comment:
So you added a boolean to each case, which is always true, but overridden to false in this case within the test helper for selinux. Huh. How is this helping, again? 😆 

As currently constructed, the `if` within the test helper is perfectly fine without any external help ...

Reply:
I removed the "smarts" from the test helper and put them in the driver function, where, arguably, they belong (which leaves the helper better focused).  So, now the boolean values are actually significant.  🙂 

However, there is some quirk of Go that I'm not in touch with which prevents me from modifying just the boolean value for the "No options" scenario, so I had to arrange for a whole new `param` value for it.  😛 

(BTW, check your Slack DMs!)